app/models/location.rb

```diff
@@ -1,16 +1,20 @@
 class Location < ApplicationRecord
 
+  belongs_to :county, optional: true
   has_many :item_locations
   has_many :items, through: :item_locations
 
-  validates :name, :address, :city, :zipcode, :state, presence: true
-  
+  # validates :name, :address, :city, :zipcode, :state, presence: true
+
 
   scope :alphabetical , -> { order('name') }
-  scope :active , -> { where(active: true) }
-  scope :inactive , -> { where(active: false) }
-  scope :for_zipcode, -> (zip) { where("zip=?", zip ) }
+
```

Comment:
Why are the active and inactive scopes being deleted? I think it is important that locations be marked as inactive or active.